src/irr/video/COpenGLImage.h

```diff
@@ -37,27 +37,27 @@ class COpenGLImage final : public IGPUImage, public IDriverMemoryAllocation
 			internalFormat(GL_INVALID_ENUM), target(GL_INVALID_ENUM), name(0u)
 		{
 			#ifdef OPENGL_LEAK_DEBUG
-				COpenGLExtensionHandler::textureLeaker.registerObj(this);
+				COpenGLFunctionTable::textureLeaker.registerObj(this);
```

Comment:
you can get rid of the `textureLeaker` we will replace it with something better in the future